todo-list/src/components/List.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <div style="width: 300px">
+        <input v-model="newTodoName" placeholder="New Todo"
+               @keyup.enter="addNewTodo"/>
+        <button @click="addNewTodo" >Add</button>
+        <ListItem v-for="todo in todos" v-model="todos" :key="todo.id"
+                  :todo="todo"
+                  @on-delete-clicked="onTodoDelete"
+                  @on-name-changed="onNameChanged"
+                  @on-done-changed="onDoneChanged" >
```

Comment:
```suggestion
                  @done-changed="onDoneChanged" >
```
Learned sth. new today! I found: https://vuejs.org/v2/guide/components-custom-events.html#Event-Names I didn't know one should use kebab case for event names.
@mattwr18 @ogerly @tirokk @alina-beck :point_up: 

From what I can tell by experience though, is that most people don't prefix the event name with `on`, like `@on-delete-clicked`. You can call it `@delete-clicked`.

Reply:
We removed the on prefix in the latest commit